app/controllers/application_controller.rb

```diff
@@ -5,14 +5,14 @@
 # General controller to include all-encompassing behavior
 class ApplicationController < ActionController::API
   include ActionController::Helpers
-  include Pundit
+  include Pundit::Authorization
   include Authentication
   include ExceptionNotifierCustomData
   include Metadata
   include Pagination
   include Collection
   include Rendering
-  include Parameters
+  include Parammeters
```

Comment:
Is this a typo ?

Reply:
:laughing: no, there's a name collision and I just wanted to bypass it as fast as possible :see_no_evil: this branch is just for testing https://issues.redhat.com/browse/RHICOMPL-3218